docs/spec/spec.md

```diff
@@ -0,0 +1,310 @@
+
+1: At Least One(call, result)
+
+#### Metadata
+
+##### Owner References
+
+- If a resource controller created this Subscription: Owned by the originating
+  resource.
+
+##### Conditions
+
+- **Ready.**
+- **FromReady.**
+- **CallActive.** True if the call is sinking events without error.
+- **Resolved.**
+
+#### Events
+
+- PublisherAcknowledged
+- ActionFailed
+
+### Life Cycle
+
+| Action | Reactions                                                        | Limitations |
+| ------ | ---------------------------------------------------------------- | ----------- |
+| Create | The publisher referenced needs to be watching for Subscriptions. |             |
+| Update |                                                                  |             |
+| Delete |                                                                  |             |
+
+---
+
+## kind: EventType
+
+NOTE: EventType is out of scope for 0.1 release. This is future documentation.
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a particular schema of Events which may be produced by one or more
+source systems._
+
+### Object Schema
+
+#### Spec
+
+| Field      | Type   | Description                                         | Limitations                    |
+| ---------- | ------ | --------------------------------------------------- | ------------------------------ |
+| jsonSchema | String | The Json Schema that represents an event in flight. | Only for JSON transport types. |
+
+#### Metadata
+
+##### Owner References
+
+EventType is owned by _Provisioners_. Each _Provisioner_ creates a
+non-controlling OwnerReference on the EventType resources it knows about.
+
+#### Status
+
+| Field          | Type    | Description                         | Limitations |
+| -------------- | ------- | ----------------------------------- | ----------- |
+| referenceCount | Integer | Number of Owners for this EventType |             |
+
+##### Conditions
+
+- **Pinned**. True when the EventType cannot be deleted.
+
+#### Events
+
+- **Owned.** When EventType has a new Provisioner Owner.
+- **Released.** When a Provisioner removes Ownership from the EventType.
+
+### Life Cycle
+
+| Action | Reactions | Limitations                                |
+| ------ | --------- | ------------------------------------------ |
+| Create |           |                                            |
+| Update |           |                                            |
+| Delete |           | Blocked until all provisioners release it. |
+
+---
+
+## Shared Object Schema
+
+### ProvisionerReference
+
+| Field                | Type            | Description | Limitations                   |
+| -------------------- | --------------- | ----------- | ----------------------------- |
+| ref<sup>1</sup>      | ObjectReference |             |                               |
+| selector<sup>1</sup> | LabelSelector   |             | Must match only one resource. |
+
+1: One of (name, selector), Required.
```

Comment:
Prefer font or ❓ emoji over superscript + follow-on references. It's likely to format better and be easier to read.

Reply:
What is `selector` for? It seems like it might better be served (in the future) using something like the [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/) abstraction.

Reply:
it is future ideas